housekeeper/cli/tables.py

```diff
@@ -87,3 +89,67 @@ def get_versions_table(rows: List[dict]) -> Table:
             version_obj["expires_at"].split("T")[0] if version_obj["expires_at"] else "",
         )
     return table
+
+
+def squash_names(list_of_files):
```

Comment:
```suggestion
def squash_names(list_of_files: List[str]) -> List[str]:
```

Reply:
OK. Though it's a list of dicts:

`def squash_names(list_of_files: List[dict]) -> List[dict]:`

I'll do a patch.